src/main/java/com/google/cloud/tools/appengine/api/deploy/DefaultStageFlexibleConfiguration.java

```diff
@@ -38,6 +39,15 @@ public void setAppEngineDirectory(File appEngineDirectory) {
   }
 
   @Override
+  public boolean getIncludeOptionalConfigurationFiles() {
+    return includeOptionalConfigurationFiles;
+  }
+
+  public void setIncludeOptionalConfigurationFiles(boolean includeOptionalConfigurationFiles) {
```

Comment:
@elharo that's already supported through deployables, which is how Intellij supports this for standard.   
@chanseokoh I'm not convinced we need this at all, but additionally we shouldn't be staging these files anywhere. Staging is only required for applications.

On a tangent: I think the 'optional configuration files' terminology used here and in the Eclipse UI is a bit poor.